plugin/trino-vertica/src/main/java/io/trino/plugin/vertica/VerticaClient.java

```diff
@@ -435,4 +444,36 @@ public OptionalInt getMaxColumnNameLength(ConnectorSession session)
     {
         return this.getMaxColumnNameLengthFromDatabaseMetaData(session);
     }
+
+    private static LongWriteFunction timestampWriteFunction(TimestampType timestampType)
```

Comment:
nit: Move this method under `timestampColumnMappingUsingString`. We usually put a helper method after the caller.